packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -94,6 +94,23 @@
                     </oc-button>
                   </div>
                 </li>
+                <template v-if="getMimeTypes">
```

Comment:
Your current version seems outdated...we've renamed "getMimeTypes" a couple of days ago

Reply:
Fixed